### PR TITLE
Feature/views

### PR DESCRIPTION
**Pull request by 2004-okari**
- Implemented the design from the sneak peek wireframes.
  - Styling is nice to have but focus on making sure everything functions first.

### CODE REVIEW

File: app/views/helpers/_post.html.erb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    <div class='comments-likes'>
+      <span>
+        Comments:<%= post.comments_counter.nil? ? '0' : post.comments_counter %>
+      </span>
+      <span>
+        Likes:<%= post.likes_counter.nil? ? '0' : post.likes_counter %>
+      </span>
+    </div>   
+  </div>
+  <%= render 'helpers/comments', post: post %>
```
Comment (Qoosim) [inline app/views/helpers/_post.html.erb]: - [x] Great job using form helper for all your forms. It's really a good practice and I give you kudos for doing so. You're great man. However, there's a little mistake here, I think this should be `helpers/all_comments` not `helpers/comments`. I got this error with your present reference, but worked fine when I changed it to `helpers/all_comments`. kindly fix it.

[comment-ref.webm](https://github.com/2004-okari/BLOG_APP/assets/26597432/9d256e65-0bc8-44a0-b050-934262b00b1b)

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @2004-okari, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  Implemented some project requirements. :heavy_check_mark: 
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Gitflow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/helpers/_post.html.erb (hunk 1 above)
Comment (2004-okari) [reply app/views/helpers/_post.html.erb]: Thanks for the feedback👍👍. The issue was also bugging me

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @2004-okari    :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**